WPMedia/fusion-news-theme-blocks
# tmedia-614-tap-targets

## Description
Promo's that are children of the top table block are not providing enough of a tap target on mobile

## Jira Ticket
[- [TMEDIA-](https://arcpublishing.atlassian.net/browse/TMEDIA-)](https://arcpublishing.atlassian.net/browse/TMEDIA-614)

## Acceptance Criteria
Acceptance criteria
On all-blocks page, there should be no tap target issues between the overlapping headline and overline
This should only affect the mobile version

## Test Steps
1. Checkout this branch `git checkout tmedia-614-tap-targets`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/shared-styles,@wpmedia/extra-large-manual-promo-block,@wpmedia/extra-large-promo-block,@wpmedia/large-manual-promo-block,@wpmedia/large-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/medium-promo-block,@wpmedia/small-manual-promo-block,@wpmedia/small-promo-block,@wpmedia/video-promo-block,@wpmedia/top-table-list-block
`
3. Bring up published version of an all blocks page.
4. Run SEO on lighthouse.
5. Verify no tap target issues are occuring on any promo block either stand alone or inside a top table.
6. Ignore tap target issues for links that are in random inserted / embedded HTML.



## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x ] Confirmed all the test steps a reviewer will follow above are working.
- [x ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x ] Confirmed this PR has unit test files
  - [x ] Ran `npm run test`, made sure all tests are passing


## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
